src/components/VisitDescription/DescriptionModal.jsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Box, Button, Typography, Modal, TextField } from '@mui/material/';
+
+import { paths } from '../../config/paths';
+import { db, auth } from '../../config/firebase';
+import { doc, setDoc } from 'firebase/firestore';
+
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: { xs: '250px', sm: '400px', md: '600px', lg: '600px' },
+  bgcolor: 'background.paper',
+  boxShadow: '0px 4px 8px rgba(0, 0, 0, 0.2)',
+  p: 4,
+};
+
+export const DescriptionModal = () => {
+  const [description, setDescription] = useState('');
+  const [open, setOpen] = useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  console.log(description);
+
+  const navigate = useNavigate();
+  const user = auth.currentUser;
+
+  const handleSubmit = async (e) => {
+    await setDoc(doc(db, 'visits', user.uid), {
+      description: description,
+    })
+      .then(() => {
+        navigate(paths.aboutUs, { replace: true });
+      })
+      .catch((error) => {
+        alert(error.message);
+      });
+    const descriptionRef = doc(db, 'visits', user.uid);
+    setDoc(descriptionRef, { description: true }, { merge: true });
+  };
```

Comment:
decide to use async/awati or .then/.catch, they are two different convention, choose one

Reply:
It already looks different. We're just working on it with Małgosia, so don't pay attention to these changes for now :)